### DISCUSSION TITLE
Avoiding hacked bank accounts

### DISCUSSION THREAD

**Discussion by Berensbach** [bisq-network/bisq#6634] [Ideas]
I suggest designing a voluntary system where we can safely exchange BTC-FIAT to avoid transfers from hacked bank accounts. This could be through a verification of bank accounts or a reputation system similar to the one localbitcoins.com had. Also that sellers can apply a filter with the conditions that the buyer has to meet.

I'm from Chile and that problem is quite common. The problem is that when someone receives a transfer from a hacked bank account, the bank freezes the receipt account after the rightful owner files a complaint, sometimes they even force you to refund all the funds to unfreeze your account and not initiate a judicial case.

**Comment by pazza83** [comment]
Have you had a look at what Bisq does currently to mitigate this?

For example:

- Account signing
- Account limits
- Account aging

See: https://bisq.wiki/Account_limits

Admittedly signing and limits are only applied to more frequently traded currencies like EUR and USD.

Would be good to have more trades in Chilean Peso :)
